### PR TITLE
Enable syntax highlighting in error stacks

### PR DESCRIPTION
**Pull request by astronomersiva**
Mostly ported from https://github.com/broccolijs/broccoli/pull/415

Babel transpilation errors:

Screen Shot 2019-07-02 at 00 19 34

HBS compilation errors:

Screen Shot 2019-07-02 at 00 18 21

### CODE REVIEW

File: lib/templates/error.html [hunk 1]
```diff
@@ -14,9 +14,14 @@
       }
       pre, code {
         font-size: 15px;
-        color: #865931;
+        color: #EFEBEA;
```
Comment (astronomersiva) [inline lib/templates/error.html:17]: Color codes were taken from https://emberjs.com/images/brand/Ember-Brand-Guidelines.pdf

File: lib/templates/error.html [hunk 2]
```diff
@@ -1091,31 +1102,34 @@ <h1 class="error-header">Dang! Looks like a {{errorType}}.</h1>
           <h2 class="error-message">{{location.file}}{{#if location.line}}:{{location.line}}{{/if}}</h2>
         {{/if}}
 
-        {{#if codeFrame}}
-          <pre><code class="javascript">{{codeFrame}}</code></pre>
-        {{/if}}
+          {{#if codeFrame}}
+            <pre><code class="javascript">{{#if hasAnsi}}{{{codeFrame}}}{{else}}{{codeFrame}}{{/if}}</code></pre>
```
Comment (astronomersiva) [inline lib/templates/error.html:1106]: Is there a better way to do this?

**Comment by astronomersiva** [comment]
@stefanpenner please take a look when you are free.

**Comment by astronomersiva** [comment]
@stefanpenner, please take a look at this if you are free.

**Comment by stefanpenner** [comment]
looking now!

File: lib/templates/error.html (hunk 2 above)
Comment (stefanpenner) [reply lib/templates/error.html:1106]: not that I can think of this looks good to me!

**Review by stefanpenner — APPROVED** [review]

**Comment by stefanpenner** [comment]
looks awesome, thank you!

**Pull request merged**